webrtc/rtcpeerconnection/rtcpeerconnection-constructor.html

```diff
@@ -0,0 +1,136 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCPeerConnection constructor</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+test(function() {
+  assert_equals(RTCPeerConnection.length, 0);
+}, 'RTCPeerConnection.length');
+
+const toStringThrows = { toString: function() { throw new Error; } };
+const toNumberThrows = Symbol();
+
+const argTests = {
```

Comment:
Shouldn't this be called "testArgs", since it's not the tests, it's the args?

It's nice to have a comment here saying that the name is the argument to pass, and the value is either false (if it's expected to succeed) or the error that is supposed to be thrown. Without it, understanding that false means success requires reading the code in detail.